### ISSUE TITLE
Feedback and suggestions

### ISSUE THREAD

**Issue by Nerders** [redis/RedisDesktopManager#5223]
Sorry, due to the general English ability, you can only submit this Issue with the help translation software.

First of all, the experience is more and more friendly after upgrade, and thank the developers again!
Follow it, the problem of recently discovered.
as the picture shows,
In Windows's dark mode, the title bar of the software main page is adapted to have a dark mode, but at the window after the settings, you can see the title bar is light color mode, this sudden look, a bright effect, let People are very uncomfortable, although there are many uses the window's usage is much relative to the homepage, but it is also very hoped to handle this problem.
Finally, I hope to add an option to adjust the list on the left side of the main page.
As seen, I only have two Redis servers, but it looks so compact on the left, which makes people exist in the process of eruption, and feel in use.
In short, Resp.App is a good software, the above two points are where I have encountered experience in experience in my recent use, and I hope that I can get it. thanks

**Comment by uglide** [comment]
> Finally, I hope to add an option to adjust the list on the left side of the main page.
As seen, I only have two Redis servers, but it looks so compact on the left, which makes people exist in the process of eruption, and feel in use.

Could you please explain this in more detail with screenshots? You can use Chinese it's totally ok.

**Comment by Nerders** [comment]
> 引用到
抱歉，忘记放截图了
![image](https://user-images.githubusercontent.com/56071530/161177089-e40c2270-de56-4dbd-95f5-0fb9e6773123.png)

**Comment by uglide** [comment]
@Nerders Left tree can be hidden using <img width="23" alt="image" src="https://user-images.githubusercontent.com/1655867/163738793-d77c5e23-3ecb-4147-92ab-e91367514c40.png"> button. The second problem with window titlebars is tracked here #5133
